src/actions/mcode.js

```diff
@@ -58,7 +59,8 @@ function processSimilarPatientOutcomes() {
     return (dispatch, getState) => {
         const { similarPatientProps } = getState().mcode;
         const service = new ServiceManager().getService('outcomes');
-        return service.processSimilarPatientOutcomes(similarPatientProps).then(results => {
+        return service.processSimilarPatientOutcomes(similarPatientProps).then(unprocessedResults => {
```

Comment:
It's a little strange to see a function `processSimilarPatientOutcomes` get called, and then what it returns is called `unprocessedResults`. Can the names be clarified here?